packages/cli/src/cli/config/action.import.ts

```diff
@@ -17,7 +17,7 @@ import fetch from 'node-fetch';
 import { invalidateCache } from '../util.js';
 import { Q, Updater } from './config.update.js';
 import { FeatureCollection } from 'geojson';
-import { CogJobJson } from '../../cog/types.js';
+import { CogJobJson } from '@basemaps/shared/build/cog/types.js';
```

Comment:
We should export this from `@basemaps/shared` we should avoid the imports from /build unless we have a really good reason for them